.github/workflows/release-kotlin-bindings.yml

```diff
@@ -0,0 +1,127 @@
+name: Release Kotlin Bindings
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - np/update-android-bindings
```

Comment:
We can do this on main, right

Reply:
Totally. Just added this to trigger it for testing. Will remove. Thought I had it working but just made a few more tweaks.